src/inc/clrnt.h

```diff
@@ -937,10 +937,22 @@ typedef struct _DISPATCHER_CONTEXT {
 
 #ifdef _TARGET_ARM_
 #include "daccess.h"
+
+//
+// Define unwind information flags.
+//
+
+#define UNW_FLAG_NHANDLER               0x0             /* any handler */
+#define UNW_FLAG_EHANDLER               0x1             /* filter handler */
+#define UNW_FLAG_UHANDLER               0x2             /* unwind handler */
                                                             
 // This function returns the length of a function using the new unwind info on arm.
 // Taken from minkernel\ntos\rtl\arm\ntrtlarm.h.
+#ifdef _MSC_VER_
 FORCEINLINE
```

Comment:
Its a bit of a nit, but I don't like this in clrnt.h. We should add FORCEINLINE to the PAL maybe?